src/classes/ProfileResults.ts

```diff
@@ -126,11 +114,16 @@ module.exports = class ProfileResults {
    * @param {String=} options.graph
    * @param {String=} options.graph_path
    */
-  compareToSnapshot (path, options = {}) {
-    let obj;
+  public compareToSnapshot(path: string, options: {
+    logResultsDiff?: boolean,
```

Comment:
You can remove the doc comment here.

Also, I find that the following is much more readable when you are splitting arguments across lines:
```
  public compareToSnapshot(
    path: string,
    options: {
      logResultsDiff?: boolean,
      graph?: string,
      graph_path?: string
    } = {},
  ): Comparison | null {
    let obj: ResultData;
```